docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -319,17 +375,50 @@ variables; worker nodes **must** specify the `CUBESTORE_WORKER_PORT` and
 `CUBESTORE_META_ADDR` environment variables. More information about these
 variables can be found [in the Environment Variables reference][ref-config-env].
 
+A sample Docker Compose stack setting this up might look like:
+
+```yaml
+version: '2.2'
+services:
+  cubestore_router:
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_SERVER_NAME=localhost
+      - CUBESTORE_META_PORT=9999
+      - CUBESTORE_WORKERS=localhost:9001
+  cubestore_worker:
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_SERVER_NAME=localhost
+      - CUBESTORE_WORKER_PORT=9001
+      - CUBESTORE_META_ADDR=localhost:9999
+  cube:
+    image: cubejs/cube:latest
+    ports:
+      - 4000:4000
+    env_file: .env
+    depends_on:
+      - cubestore
+    links:
+      - cubestore
```

Comment:
@hassankhan This is a copy-paste typo, right? Should not be just `cubestore`, I suppose?

Reply:
Indeed it was 😀  fixed in the last push